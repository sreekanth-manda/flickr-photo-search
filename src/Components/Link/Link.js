import React from 'react';

const Link = ({ hrefText, linkText }) => (
  <a className="link" href={hrefText} target="_blank" rel="noopener noreferrer">{linkText}</a>
);

export default Link;
