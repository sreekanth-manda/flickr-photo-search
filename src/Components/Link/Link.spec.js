import React from 'react';
import { shallow } from 'enzyme';
import Link from './Link';

describe('Link', () => {
  const componentProps = {
    hrefText: 'blah href text',
    linkText: 'blah link text'
  }
  test('should render the Link component', () => {
    const wrapper = shallow(<Link { ...componentProps } />);
    expect(wrapper.html()).toMatchSnapshot();
  });
});
