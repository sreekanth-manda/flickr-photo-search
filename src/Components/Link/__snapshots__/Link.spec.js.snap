// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link should render the Link component 1`] = `"<a class=\\"link\\" href=\\"blah href text\\" target=\\"_blank\\" rel=\\"noopener noreferrer\\">blah link text</a>"`;
